modules/cnv-cdrom-wizard-fields-web.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-create-vms.adoc
+// * cnv/cnv_vm_templates/cnv-creating-vm-template.adoc
+
+[id="cnv-cdrom-wizard-fields-web_{context}"]
+= CD-ROM fields
+
+|===
+|Media Type | Description
+
+|Container
+|Specify the container path. For example: kubevirt/fedora-registry-disk: latest.
+
+|Persistent volume claim
+|Select the desired persistent volume claim from the drop-down list.
+
+|URL
+|Specify the URL path and size in GiB. Then, select the storage class for this URL from the drop-down list.
+
+|Windows guest tools
```

Comment:
"Windows guest tools" is not a CD-ROM source on wizard, the windows-guest-tools is attached automatically if it's windows. This type should be removed.

Reply:
correct

Reply:
Removed Windows guest tools.